src/decoder/ifd.rs

```diff
@@ -10,13 +10,14 @@ use tags::{Tag, Type};
 use {TiffError, TiffFormatError, TiffResult, TiffUnsupportedError};
 
 use self::Value::{
-    Ascii, Double, Float, List, Rational, RationalBig, SRational, SRationalBig, Signed, SignedBig,
+    Ascii, Byte, Double, Float, List, Rational, RationalBig, SRational, SRationalBig, Signed, SignedBig,
     Unsigned, UnsignedBig,
 };
 
 #[allow(unused_qualifications)]
 #[derive(Debug, Clone, PartialEq)]
 pub enum Value {
+    Byte(u8),
```

Comment:
Note how `short` is implemented, using `Unsigned`. We will want to change this at some point but for now please use the same style.

Reply:
I don't get this. Lets say I remove `Byte` from `Value` and change the part in the `val()` function to:

```rust
                (Type::UNDEFINED, n) => self.decode_offset(n, bo, limits, decoder, |decoder| {
                    offset_to_bytes(8, self)
                }),
```
I would still need the `into_u8()` and `into_u8_vec()` but with what type?

Reply:
Actually it's fine. I think we'll want to rework the internals of `Value` for the next major release so this shouldn't affect anything. Keep it the way it is :slightly_smiling_face:

Reply:
Okay!